src/server/pfs/db/driver.go

```diff
@@ -125,12 +126,28 @@ func initDB(session *gorethink.Session, dbName string) error {
 		return nil
 	}
 
+	// There is a race here
+	//
+	// When rethink is under load, we consistently fail on the table creation
+	// w a 'db not found' error right after we created it a few lines above.
+	//
+	// However, the db shows up fine in the dashboard, so it does in fact get
+	// created. This is only relevant for tests, because thats the only time
+	// we create a bunch of databases while R/W from rethink as well
+	config := backoff.NewExponentialBackOff()
+	// We want to backoff more aggressively (i.e. wait longer) than the default
+	config.InitialInterval = 1 * time.Second
+	config.Multiplier = 2
+	config.MaxElapsedTime = 5 * time.Minute
 	// Create tables
 	for _, table := range tables {
-		tableCreateOpts := tableToTableCreateOpts[table]
-		if _, err := gorethink.DB(dbName).TableCreate(table, tableCreateOpts...).RunWrite(session); err != nil {
+		backoff.RetryNotify(func() error {
+			tableCreateOpts := tableToTableCreateOpts[table]
+			_, err := gorethink.DB(dbName).TableCreate(table, tableCreateOpts...).RunWrite(session)
 			return err
-		}
+		}, config, func(err error, d time.Duration) {
+			fmt.Printf("Error; retrying in %s: %#v\n", d, err)
```

Comment:
Last little tweak, this should log using `lion` instead of `fmt` to match the rest of our code base. Also it'd be good to describe where the error is coming from, ie `Error creating table`. Also using `%s` and `err.Error()` instead of `%#v` might be a good idea but those might also be equivalent. I'm not quite sure.